tests/unit/test_paging.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import json
+
+from tests.framework import (CapturedIOTestCase)
+from globus_sdk.transfer.paging import PaginatedResource
+from globus_sdk.transfer.response import IterableTransferResponse
+
+
+class PagingSimulator():
```

Comment:
Should be `class PagingSimulator(object):`
Otherwise, this will be an old-style class. Unlikely to be a problem, but good to fix for cleanliness.